File: python/nivel 2 logica/11.py
"""
Ejercicio 11
Determinar si un número es primo
"""
numero = int(input("Ingrese el Numero: "))

if numero <= 1:
  print("El numero no es Primo")
else:
  es_primo = True

  for i in range(2, numero):
    if numero % i == 0:
      es_primo = False
      break

  if es_primo:
    print(numero," Es un numero primo")
  else:
    print(numero," No es un numero primo")

File: javascript/nivel 2 logica/11.js
/*
Ejercicio 11
Determinar si un número es primo
*/

let numero = parseInt(prompt("Ingrese un numero: "));

if(numero <= 1){
  console.log("el numero no es primo");
}else{
  let esPrimo = true;

  for (let i = 2; i < numero; i++) {
    if(numero % i === 0){
      esPrimo = false;
      break
    }
  }
  if(esPrimo){
    console.log("El numeros ",numero," Es primo");
  }else{
    console.log("El numeros ",numero," No es primo");
  }
}